harness/tests/tensorboard/test_base.py

```diff
@@ -30,7 +30,7 @@ def test_setting_optional_variable(tmp_path: pathlib.Path) -> None:
         env.det_cluster_id, env.det_experiment_id, env.det_trial_id, checkpoint_config
     )
     assert isinstance(manager, tensorboard.SharedFSTensorboardManager)
-    assert manager.base_path == pathlib.Path("test_value/tensorboard")
+    assert manager.base_path == pathlib.Path("test_value/tensorboard-")
```

Comment:
what is this for?

Reply:
The tensorboard path is now of form f"{tensorboard}-{allocation_id}" or f"{tensorboard}-{allocation_id}-{rank}". As you can see, at least one dash is now there.